content-repo/extra-docs/articles/Email-Communication-Readme.md

```diff
@@ -0,0 +1,195 @@
+
+* **EWS - Classifier - Email Communication**:  Classifies EWS email messages.
+* **EWS - Incoming Mapper - Email Communication**:  Maps incoming EWS email message fields. 
+* **Gmail - Classifier - Email Communication**:   Classifies Gmail email messages. 
+* **Gmail - Incoming Mapper - Email Communication**: Maps incoming Gmail email message fields.
+
+### Incident Fields
+There is 1 incident field - **Add CC To Email**. 
+
+### Incident Types
+There is 1 incident type - **Email Communication**.
+
+### Layout
+There is 1 layout - **Email Communication** 
+
+There are 3 interactive sections in which you can specify 1 or more email addresses to add as a CC to the email, create the body of the email, and add attachments. The remaining 3 sections are for viewing the original email and all the email communications and attachments associated with the incident.
+
+
+
+![Layout](https://raw.githubusercontent.com/demisto/content/84e7bc89c8757544804540e6711d4b9aba210ec1/Packs/EmailCommunication/doc_files/Email_Communication_layout.png)
+
+You can use the layout as-is for email communication. It can also be used for new incident types by adding the **Email Communication** layout to an incident type. See [Add the Email Communication Layout to an Incident Type](#add-the-email-communication-layout-to-an-incident-type) for details. 
+ 
+>**Important:** 
+- In order to add CC recipients or an attachment to the email reply, you must select the *Show empty fields* checkbox. 
+- You need to customize the *service_mail* parameter in the **Send Reply** button with the mailbox from which emails are sent. See **Customize the Send Reply Button** below.
+
+
+| Layout sections | Description |
+|------------------ | ------------- |
+| Add CC to email | Add 1 or more CC recipients to the email as a comma-separated list of email addresses.  |
+| Message body | Write the body of the email reply. It is sent as an email and added to the War Room entry. |
+| Attachments | Add attachments to the email reply.  |
+| Email thread | Displays the entire email thread including the original email and all email replies. |  
+| Original Email HTML | Displays in HTML format the original email that opened the incident. |  
+| Mail Attachments | Displays the metadata of the email attachments. Contains the Send Reply button.  |  
+ ---
+
+> Note: If an email cannot be sent to a specified address, no notification will appear in Cortex XSOAR.
+ 
+## Before You Start
+
+This pack requires that you must have active instances of both a mail listener and mail sender integration in order to send and receive emails. Configure either the [Gmail integration](https://xsoar.pan.dev/docs/reference/integrations/gmail)  or both the [EWS Mail Sender](https://xsoar.pan.dev/docs/reference/integrations/ews-mail-sender) and [EWS V2](https://xsoar.pan.dev/docs/reference/integrations/ews-v2) integrations. In addition, configure the Demisto REST API integration which requires a Demisto API key.
```

Comment:
```suggestion
This pack requires that you must have active instances of both a mail listener and mail sender integration in order to send and receive emails, and an active instance of Demisto REST API integration. Configure either the [Gmail integration](https://xsoar.pan.dev/docs/reference/integrations/gmail) or both the [EWS Mail Sender](https://xsoar.pan.dev/docs/reference/integrations/ews-mail-sender) and [EWS V2](https://xsoar.pan.dev/docs/reference/integrations/ews-v2) integrations. In addition, configure the Demisto REST API integration which requires a Demisto API key.
```

need xref for Demisto REST API docs

Reply:
There no Rest API docs yet, but we added an explanation of how to configure the demisto rest api